src/ride/track.c

```diff
@@ -4921,15 +4922,446 @@ void game_command_remove_track(int *eax, int *ebx, int *ecx, int *edx, int *esi,
 	return;
 }
 
+money32 loc_6CDDFD() {
+    // pop     edi
+    // pop     edx
+    // pop     ecx
+    // pop     ebx
+    // pop     eax
+    // mov     ebx, dword_F4413E
+
+    // test    park_flags, 800h
+    // jz      short locret_6CDE16
+    // xor     ebx, ebx
+    // locret_6CDE16:
+    // retn
+    if (RCT2_GLOBAL(RCT2_ADDRESS_PARK_FLAGS, uint32) & PARK_FLAGS_NO_MONEY) {
```

Comment:
Just inline this into the code in the 3 places it is used as its a pretty small and easy to understand bit of code.